scripts/setupPythonEnvironment.bash

```diff
@@ -19,42 +19,42 @@ declare -a LINK_SCRIPTS=("preprocess_xml"
 
 
 # Read input arguments
-PYTHON_TARGET="$(which python)"
-VIRTUAL_PATH=""
-VIRTUAL_NAME="geosx"
-MINICONDA_BUILD=""
-INSTALL_VIRTUAL=false
-BIN_DIR=""
+PYTHON_TARGET=
+BIN_DIR=
+PIP_CMD="pip --disable-pip-version-check"
+
+
+if [[ -z "${VERBOSE}" ]]
+then
+    VERBOSE=false
+else
+    VERBOSE=true
+fi
+
 
 while [[ $# > 0 ]]
 do
 key="$1"
 
 case $key in
-    -p|--python_target)
+    -p|--python-target)
     PYTHON_TARGET="$2"
     shift # past argument
     ;;
-    -m|--miniconda_build)
-    MINICONDA_BUILD="$2"
-    shift # past argument
-    ;;
-    -v|--virtual_path)
```

Comment:
When running `cmake`, there's still an error message complaining about `virtualenv` not being installed (for `pygeos`). Do you think it's still relevant?  (I do not know what it's about)

Reply:
This check shows up on this line:

https://github.com/GEOS-DEV/GEOS/blob/develop/src/coreComponents/python/CMakeLists.txt#L3-L20

As this warning suggests, building `pygeos` requires that the target environment have the virtualenv package installed.  As a final step for this PR, I'll move this check inside the `ENABLE_PYGEOSX` check and change it to an error.